src/utils/matchers/li-username-matcher.ts

```diff
@@ -0,0 +1,4 @@
+export const getLinkedInUsername = (url: string) => {
```

Comment:
grouping all scrapers in one file (the earlier one) would be better, instead of having multiple files with just one function.

Reply:
Right. I'll move the matcher's to single file and export them from there. Separate files do seem unnecessary.

Reply:
@diivi, as you proposed, I've moved the matchers to a single utility file and updated the file-names to stay consistent with the others.
c7c72ef10533a844160c23a95ef28310fe46832b